packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -302,6 +302,15 @@ contract Attestations is
     emit Withdrawal(msg.sender, token, value);
   }
 
+  function withdrawToSigner(address token) external {
+    address issuer = getAccounts().attestationSignerToAccount(msg.sender);
+    uint256 value = pendingWithdrawals[token][issuer];
+    require(value > 0, "value was negative/zero");
+    pendingWithdrawals[token][issuer] = 0;
+    require(IERC20(token).transfer(msg.sender, value), "token transfer failed");
```

Comment:
@nategraf Suggested that we should move the funds to the `issuer` to maintain our attempt at making the account  key the one that can move balances, and not the signers